deta/base.py

```diff
@@ -91,7 +89,35 @@ def _request(self, path: str, method: str, data: dict = None):
 
         if status in [200, 201, 207, 404]:
             return status, json.loads(payload) if status != 404 else None
-        raise urllib.error.HTTPError(url, status, res.reason, res.headers, res.fp)
+        raise urllib.error.HTTPError(
+            url, status, res.reason, res.headers, res.fp)
+
+class Drive(_Object):
+    def __init__(self, drive_name:str=None, project_key:str=None, project_id:str=None, host:str=None):
+        assert drive_name, "Please provide a Drive name. E.g 'mydrive"
+        assert project_key, "Please provide a project_key. Check docs.deta.sh"
+        self.name = drive_name
+        self.project_key = project_key
+        self.project_id = project_id
+        host = host or os.getenv("DETA_DRIVE_HOST") or "drive.deta.sh"
+        self.client = http.client.HTTPSConnection(host, timeout=3)
+        self.base_path = "/v1/{0}/{1}".format(self.project_id, self.name)
+        self.util = Util()
+
+
+
+class Base(_Object):
```

Comment:
same here, now that I look at it, even the `base_path` is the same how it's constructed for both the services. You can move all the `init` logic to the parent class and only have `util` for `Base`